packages/charts/src/LineChart/LineChart.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import PropTypes from 'prop-types';
+import PropTypes, { any } from 'prop-types';
```

Comment:
This overwrites the built-in `any` type. Did you mean that?

Reply:
Wrong auto import, will change it